package.json

```diff
@@ -61,7 +61,7 @@
     "minimatch": "^10.0.1",
     "mongodb": "^6.11.0",
     "node-forge": "^1.3.1",
-    "node-rdkafka": "^2.12.0",
+    "node-rdkafka": "^3.4.1",
```

Comment:
Could we test a zenko build with this change early, as this is a major version bump?